added payout addresses balance (#284)
dashpay/docs: docs/user/network/dashmate/index.rst

-----------------------

Copy the provided protx command and run it using dash-cli or the Dash Core

> console. Do note that your payout addresses must have some balance for the
> registration process to be successful. Hence, remember to send some DASH tokens
> to these addresses before you begin registration.

Reviewer: ```suggestion
console. Do note that your payout address must have a balance for the
registration process to be successful, so remember to send some DASH
to this address before you begin registration.
```
